dds/DCPS/MessageBlock.h

```diff
@@ -0,0 +1,49 @@
+#ifndef OPENDDS_DCPS_MESSAGEBLOCK_H
+#define OPENDDS_DCPS_MESSAGEBLOCK_H
+
+#if !defined (ACE_LACKS_PRAGMA_ONCE)
+#  pragma once
+#endif /* ACE_LACKS_PRAGMA_ONCE */
+
+#include "dcps_export.h"
+
+#include <dds/Versioned_Namespace.h>
+
+#include <ace/Message_Block.h>
+
+OPENDDS_BEGIN_VERSIONED_NAMESPACE_DECL
+
+namespace OpenDDS {
+namespace DCPS {
+
+struct OpenDDS_Dcps_Export MessageBlock {
```

Comment:
This used to be nested inside another class, so there wasn't a need to make data private.  Now it should be an actual `class` with private member data.